internal/rootcoord/dml_channels_test.go

```diff
@@ -157,6 +158,38 @@ func TestDmlChannels(t *testing.T) {
 
 	dml.removeChannels(chans0...)
 	assert.Equal(t, 0, dml.getChannelNum())
+
+	paramtable.Get().Save(Params.CommonCfg.PreCreatedTopicEnabled.Key, "true")
+	paramtable.Get().Save(Params.CommonCfg.TopicNames.Key, "topic1,topic2")
+	defer paramtable.Get().Reset(Params.CommonCfg.PreCreatedTopicEnabled.Key)
+	defer paramtable.Get().Reset(Params.CommonCfg.TopicNames.Key)
+
+	dml1 := newDmlChannels(ctx, factory, dmlChanPrefix, totalDmlChannelNum)
+	chanNames1 := dml1.listChannels()
+	assert.Equal(t, 0, len(chanNames1))
```

Comment:
Don't get how this can pass, according to the code, it should panic at line 167 because topic1, topic2 don't both exist